app/DynamicsAdapter/DynamicsAdapter.Web/PersonSearch/PersonSearchController.cs

```diff
@@ -44,6 +44,16 @@ public class Person
     }
 
 
+    public class ProviderSearchEventStatus
+    {
+        public Guid SearchRequestId { get; set; }
```

Comment:
I think here should be SearchApiRequestId.

Reply:
no, that's a concept from Search Api app, SearchApiRequestId is a dynamics concept

Reply:
ok. that's fine. So, this SearchRequestId in actually the SearchApiRequestId from Dynamics.
people will get confused about SearchRequest and SearchApiRequest.